engines/config-query-sparql/config/rdf-dereference/actors.json

```diff
@@ -2,24 +2,17 @@
   "@context": [
     "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/runner/^2.0.0/components/context.jsonld",
 
-    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-rdf-dereference-http/^2.0.0/components/context.jsonld",
-    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-rdf-dereference-fallback/^2.0.0/components/context.jsonld"
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/bus-rdf-dereference/^2.0.0/components/context.jsonld"
   ],
   "@id": "urn:comunica:default:Runner",
   "@type": "Runner",
   "actors": [
     {
-      "@id": "urn:comunica:default:rdf-dereference/actors#http",
-      "@type": "ActorRdfDereferenceHttp",
-      "beforeActors": { "@id": "urn:comunica:default:rdf-dereference/actors#fallback" },
-      "mediatorHttp": { "@id": "urn:comunica:default:http/mediators#main" },
-      "mediatorRdfParseMediatypes": { "@id": "urn:comunica:default:rdf-parse/mediators#mediaType" },
-      "mediatorRdfParseHandle": { "@id": "urn:comunica:default:rdf-parse/mediators#parse" }
-    },
-
-    {
-      "@id": "urn:comunica:default:rdf-dereference/actors#fallback",
-      "@type": "ActorRdfDereferenceFallback"
+      "@id": "urn:comunica:default:rdf-dereference/actors#",
+      "@type": "ActorRdfDereference",
```

Comment:
Can this bus (and package) be renamed to `ActorDereferenceRdf`?
This would be better aligned with the naming convention of `bus + actor type`.

The `@id` could then become `urn:comunica:default:dereference-rdf/actors#parse`.

Reply:
Ah - I see the confusion here - what is happening here, is that I have a `bus-rdf-dereference` which already has `run` and `test` methods - so this bus is also being used as an actor here (since it only one actor makes sense here).

We could create a dummy actor if we wanted
```ts
class ActorDereferenceRdfMain extends ActorDereferenceRdf {
    constructor(...) {
       super(args)
    }
}
```
I figured that this was working since the integration test with the build engine worked, but I can double check this.

Reply:
That said the bus is probably better named as `bus-derference-rdf` anyway so I am happy to make that change

Reply:
+1 on the renaming.

We should keep the logic in the buses as generic as possible, since someone might want to implement an rdf-dereference actor that does not depend on the rdf-parse bus.
Therefore, I would suggest making the abstract actor in `bus-rdf-dereference` as basic as possible, and have a new actor `actor-rdf-dereference-parse` (similar to your `ActorDereferenceRdfMain`), which actually implements the wiring with `bus-rdf-parse`.
Do you think this makes sense?